tests/test_model_config_generator.py

```diff
@@ -41,11 +41,46 @@ def test_direct_no_params(self):
             """)
 
         expected_configs = [
-            {'dynamic_batching': {}, 'instance_group': [{'count': 1, 'kind': 'KIND_GPU'}]},
-            {'dynamic_batching': {}, 'instance_group': [{'count': 2, 'kind': 'KIND_GPU'}]},
-            {'dynamic_batching': {}, 'instance_group': [{'count': 3, 'kind': 'KIND_GPU'}]},
-            {'dynamic_batching': {}, 'instance_group': [{'count': 4, 'kind': 'KIND_GPU'}]},
-            {'dynamic_batching': {}, 'instance_group': [{'count': 5, 'kind': 'KIND_GPU'}]},
+            {'dynamic_batching': {}, 'max_batch_size': 1, 'instance_group': [{'count': 1, 'kind': 'KIND_GPU'}]},
```

Comment:
Should we add a utility function to generate this dynamically?  
These types of configs are in multiple files